src/components/sidemenu/SideMenuPanel.js

```diff
@@ -122,10 +122,17 @@ const getMenuItems = ({ API, hideSidemenu, showDialog, navigation, store, theme
               onPress: async () => {
                 store.set('loadingIndicator')({ loading: true })
                 const userStorage = await import('../../lib/gundb/UserStorage').then(_ => _.default)
+                let token = await userStorage.getProfileFieldValue('w3Token')
+
+                if (!token) {
+                  token = await userStorage.getProfileFieldValue('loginToken')
```

Comment:
Both login token and w3 token can be used to make w3 api calls?

Reply:
@sirpy yes. Sergey added this possibility in the scope of task about updating the w3 site with has_wallet: false while deleting wallet account.